UlKit/UlKit/ModificatorControl.swift

```diff
@@ -0,0 +1,83 @@
+// ModificatorControl.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Компонент в виде квадратной кнопки с картинкой и текстом
+final class ModificatorControl: UIView {
+    // MARK: - Visual Components
+
+    private lazy var imageView: UIImageView = {
```

Comment:
```suggestion
    private lazy var imageView: UIImageView = {
```
imageView общее название